test/simpleTest.js

```diff
@@ -18,18 +18,29 @@ describe("simple", function() {
 	test("simple2", ".class { a: b c d; }\n.two {}", [
 		[1, ".class { a: b c d; }\n.two {}", ""]
 	]);
+	test("escape characters (uppercase)", ".class { content: \"\\F10C\" }", [
+		[1, ".class { content: \"\\F10C\" }", ""]
+	]);
+	// Need uncomment after resolve https://github.com/css-modules/postcss-modules-local-by-default/issues/108
```

Comment:
@michael-ciniawsky i think best note is not in code, some issue can resolve in a long time :sob: Be good merge this as is and if some submit about problem related to this add link to issue in other repo maybe it is help solve the problem faster